fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/NumericArithmetic.java

```diff
@@ -596,4 +650,515 @@ public static Expression divideDecimalV3(DecimalV3Literal first, DecimalV3Litera
         return new DecimalV3Literal(DecimalV3Type.createDecimalV3TypeLooseCheck(
                 t1.getPrecision(), t1.getScale() - t2.getScale()), result);
     }
+
+    /**
+     * coalesce
+     */
+    @ExecFunction(name = "coalesce")
+    public static Expression coalesce(Literal first, Literal... second) {
+        if (!(first instanceof NullLiteral)) {
+            return first;
+        }
+        for (Literal secondLiteral : second) {
+            if (!(secondLiteral instanceof NullLiteral)) {
+                return secondLiteral;
+            }
+        }
+        return first;
+    }
+
+    /**
+     * round
+     */
+    @ExecFunction(name = "round")
+    public static Expression round(DecimalV3Literal first) {
+        return first.round(0);
+    }
+
+    /**
+     * round
+     */
+    @ExecFunction(name = "round")
+    public static Expression round(DecimalV3Literal first, IntegerLiteral second) {
+        if (second.getValue() >= first.getValue().scale()) {
+            return first;
+        }
```

Comment:
this `if` statement should move into `DecimalV3Literal#round`

Reply:
done